tests/unit/project_loader/extensions/test_gnome_3_34.py

```diff
@@ -0,0 +1,425 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from testtools.matchers import Equals, Contains
+
+from snapcraft.internal.project_loader._extensions.gnome_3_34 import ExtensionImpl
+
+from .. import ProjectLoaderBaseTest
+import textwrap
+
+from tests.unit.commands import CommandBaseTestCase
+
+
+class ExtensionTest(ProjectLoaderBaseTest, CommandBaseTestCase):
+    def test_extension(self):
+        gnome_extension = ExtensionImpl(
+            extension_name="gnome-3.34", yaml_data=dict(base="core18")
+        )
+
+        self.expectThat(
+            gnome_extension.root_snippet,
+            Equals(
+                {
+                    "plugs": {
+                        "gtk-3-themes": {
+                            "interface": "content",
+                            "target": "$SNAP/data-dir/themes",
+                            "default-provider": "gtk-common-themes",
+                        },
+                        "icon-themes": {
+                            "interface": "content",
+                            "target": "$SNAP/data-dir/icons",
+                            "default-provider": "gtk-common-themes",
+                        },
+                        "sound-themes": {
+                            "interface": "content",
+                            "target": "$SNAP/data-dir/sounds",
+                            "default-provider": "gtk-common-themes",
+                        },
+                        "gnome-3-34-1804": {
+                            "interface": "content",
+                            "target": "$SNAP/gnome-platform",
+                            "default-provider": "gnome-3-34-1804",
+                        },
+                    },
+                    "environment": {
+                        "SNAP_DESKTOP_RUNTIME": "$SNAP/gnome-platform",
+                        "GTK_USE_PORTALS": "1",
+                    },
+                    "layout": {
+                        "/usr/bin/gjs": {"symlink": "$SNAP/gnome-platform/usr/bin/gjs"},
```

Comment:
seems you missed this (maybe others too)

Reply:
good catch! all gjs mentions should have been cleaned up now.